MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -31,145 +32,71 @@ It is also important to note that while this document provides the criteria agai
-* Front-end modules: builds are Node 16/Yarn 1
-
-## Module is secure
-* All API endpoints protected with appropriate permissions
-* No excessive permissions granted to the module
-* Personal data form is completed, accurate, and provided as PERSONAL_DATA_DISCLOSURE.md file
-* Sensitive information is not checked into git repository
-* Tenant data is segregated at the storage layer
-* Back-end modules don’t access data in DB schemas other than their own and public
-* Tenant data is segregated at the transit layer
-* sonarqube hasn't identified any security issues
-
-## Module is multi-tenant
-* Tenant data is segregated at the storage layer
-* Back-end modules don’t access data in DB schemas other than their own and public
-* Tenant data is segregated at the transit layer
-* Back-end modules respond with a tenant’s content based on x-okapi-tenant header
-
-## Module is internationalized
-* Front-end modules have i18n support via react-intl and an en.json file with English texts
-
-## Module is meets current accessibility requirements
-* Front-end modules have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension
-
-## Module offers a cohesive user experience consistent with the rest of FOLIO
-* Front-end modules use the current version of Stripes
-* Front-end modules follow relevant existing UI layouts, patterns and norms
-* Front end modules must work in the latest version of Chrome
-
-## Module has developer and end-user documentation
-* Environment vars are documented in the ModuleDescriptor
-* API endpoints are all documented in RAML or OpenAPI
-* Installation documentation included
-
-## Module depends ONLY on other modules and infrastructure that are already included in FOLIO
-* Module only uses FOLIO interfaces already provided by previously accepted modules 
-* Module only uses existing infrastructure / platform technologies
-* Integration with any third party system (outside of the FOLIO environment) tolerates the absence of configuration / presence of the system gracefully
-
-## Module has a long-term development and maintenance plan
-* Upon acceptance, code author(s) agree to have source code canonically in folio-org github
-* Copyright assigned to OLF
-* Contribution guide is included in repo
-* Module is written in a language and framework that FOLIO development teams are familiar with
-* For backend modules: builds are Maven and JDK 11 based and Dockerfile provided
-* Front-end modules: builds are Node 16/Yarn 1
-
-## Module is scalable
-* HA compliant
-
-## Module supports high availability
-* HA compliant
+* Module gracefully handles the absence of third party systems or related configuration. (3, 5, 12)
+* Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication (6)
+* Builds with tools found on the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) list (3, 5, 13)
```

Comment:
This seems to overlap heavily with this criteria:
> Module is written in a language and framework that FOLIO development teams are familiar with _e.g. Vertx/RMB, Spring Way/folio-spring-base, and React/Stripes_ (13)

How are they different?

Reply:
One is about frameworks, and one is about build tools.  I think it would be problematic to introduce a backend module that doesn't have a pom file, but instead has an ant file, or something else.

Reply:
I'm aware of gradle, but was unaware that ant is in use.

Reply:
Although, ant is defined in our [Jenkins pipelines](https://github.com/folio-org/pipelines-shared-library/blob/b1d2b94519393070617c52c5eba556e62a978946/jenkinsResources/dependecies.json#L410)